MainWindow.xaml.cs

```diff
@@ -230,7 +268,8 @@ private void BrowseFolder()
             // If we have a custom configuration, we need to extract the best possible information from it
             if (systemName == "Custom Input" && system == KnownSystem.NONE && type == DiscType.NONE)
             {
-                Utilities.Validation.DetermineFlags(customParameters, out type, out system, out string letter, out string path);
+                string letter, path;
+                Utilities.Validation.DetermineFlags(customParameters, out type, out system, out letter, out path);
```

Comment:
I know you had problems with the older version of VS and this, but can you put this back to inline variable declaration? It's much cleaner that way.